src/bin/travis/resources/jes_centaur.conf.ctmpl

```diff
@@ -0,0 +1,71 @@
+{{with $refreshToken := vault (printf "secret/dsde/cromwell/common/refresh-token")}}
```

Comment:
No. `jes_centaur.conf` replaces the `jes.conf` in `jesConf.tar.enc`.

Cromwell runs inception like, with an outer cromwell running an inner cromwell:
1. Outer cromwell uses `jes_centaur.conf`
2. Inner cromwell uses `mulitBackend.conf`